reconcile/utils/openshift_resource.py

```diff
@@ -63,7 +75,18 @@ def obj_intersect_equal(self, obj1, obj2):
                     pass
                 elif self.ignorable_key_value_pair(obj1_k, obj1_v):
                     pass
-                elif obj1_k in ["data", "labels", "matchLabels"]:
+                elif obj1_k == "labels":
```

Comment:
is there any risk we have labels out of metadata ?

Reply:
It was like this before so probably not an issue

Reply:
the logic for labels is still the same with the exception of the `IGNORABLE_DATA_FIELDS` `service-ca.crt` but is usually within the data section of a secret and not within a label.